### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by mauri-melato**
### Description of changes
* Reviewed the changelog item list and wording.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,42 +5,43 @@ CHANGELOG
...
-  - Add new configuration parameter `Scheduling/SlurmSettings/EnableMemoryBasedScheduling` to configure memory-based scheduling in Slurm.
+- Add support for memory-based job scheduling in Slurm
+  - Configure compute nodes real memory in the Slurm cluster configuration.
+  - Add new configuration parameter `Scheduling/SlurmSettings/EnableMemoryBasedScheduling` to enable memory-based scheduling in Slurm.
   - Add new configuration parameter `Scheduling/SlurmQueues/ComputeResources/SchedulableMemory` to override default value of the memory seen by the scheduler on compute nodes.
+- Improve flexibility on cluster configuration updates to avoid the stop and start of the entire cluster whenever possible.
+  - Add new configuration parameter `Scheduling/SlurmSettings/QueueUpdateStrategy` to set the preferred strategy to adopt for compute nodes needing a configuration update and replacement.
+- Add support to mount existing FSx for ONTAP and FSx for OpenZFS file systems.
+- Add support to mount multiple instances of existing EFS, FSx for Lustre / for ONTAP/ for OpenZFS file systems.
+- Add support for FSx Lustre Persistent_2 deployment type.
```
Comment (charlesg3) [inline CHANGELOG.md]: nit: might clarify that this is for created filesystems.

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,42 +5,43 @@ CHANGELOG
...
+- Remove support for Python 3.6.
 
 **BUG FIXES**
-- Fix default for disable validate and test components when building custom AMI. The default was to disable those components, but it wasn't effective.
-- Handle corner case in the scaling logic when instance is just launched and the describe instances API doesn't report yet all the EC2 info.
-- Dropped validation that would prevent ARM instance type to be used when `DisableSimultaneousMultithreading` was set to true.
-- Add missing policies for EcrImageDeletionLambda and ImageBuilderInstance roles that were causing failure when upgrading ParallelCluster API from one version to another.
+- Fix the default behaviour to disable the validation and test components when building a custom AMI.
+- Handle corner case in the scaling logic when an instance is just launched and the describe instances API call doesn't report all the EC2 info yet.
+- Fixed support for `DisableSimultaneousMultithreading` parameter on instance types with ARM processors.
```
Comment (charlesg3) [inline CHANGELOG.md]: nit `ARM` -> `Arm`

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,42 +5,43 @@ CHANGELOG
...
 - Prompt user to enable EFA for supported instance types when using `pcluster configure` wizard.
-- Change default EBS volume types from gp2 to gp3 in both the root and additional volumes.
 - Add support for rebooting compute nodes via Slurm.
 
 **CHANGES**
-- Remove support for Python 3.6.
 - Upgrade Slurm to version 21.08.8-2.
-- Do not require `PlacementGroup/Enabled` to be set to `true` when passing an existing `PlacementGroup/Id`.
+- Upgrade EFA installer to version 1.17.2 
+  - ---TBC---
```
Comment (hanwen-pcluste) [inline CHANGELOG.md]: We can copy/paste from here https://github.com/aws/aws-parallelcluster-cookbook/blob/develop/CHANGELOG.md

**Comment by enrico-usai** [comment]
We have to add Multi-nic changes: https://github.com/aws/aws-parallelcluster-cookbook/pull/1485/files

**Review by enrico-usai — COMMENTED** [review]
Can you please add: 
``` 
- Fix file handle leak in `computemgtd`.
```
from:  https://github.com/aws/aws-parallelcluster/pull/4154/files
In this way we'll close it as duplicate.

File: CHANGELOG.md (hunk 1 above)
Comment (mauri-melato) [reply CHANGELOG.md]: Addressed this comment, thanks 👍

File: CHANGELOG.md (hunk 2 above)
Comment (mauri-melato) [reply CHANGELOG.md]: yep, done

File: CHANGELOG.md (hunk 3 above)
Comment (mauri-melato) [reply CHANGELOG.md]: Done, together with the rest of software updates that were missing...

**Comment by mauri-melato** [comment]
> We have to add Multi-nic changes: https://github.com/aws/aws-parallelcluster-cookbook/pull/1485/files

**Review by demartinofra — APPROVED** [review]
🚀
